flaml/automl/automl.py

```diff
@@ -2724,6 +2724,58 @@ def cv_score_agg_func(val_loss_folds, log_metrics_folds):
         )
         self._auto_augment = auto_augment
 
+        if "auto" == estimator_list:
+            if self._state.task == "rank":
+                estimator_list = ["lgbm", "xgboost", "xgb_limitdepth"]
+            elif _is_nlp_task(self._state.task):
+                estimator_list = ["transformer"]
+            elif self._state.task == TS_FORECASTPANEL:
+                estimator_list = ["tft"]
+            else:
+                try:
+                    import catboost
+
+                    estimator_list = [
+                        "lgbm",
+                        "rf",
+                        "catboost",
+                        "xgboost",
+                        "extra_tree",
+                        "xgb_limitdepth",
+                    ]
+                except ImportError:
+                    estimator_list = [
+                        "lgbm",
+                        "rf",
+                        "xgboost",
+                        "extra_tree",
+                        "xgb_limitdepth",
+                    ]
+                if self._state.task in TS_FORECAST:
+                    # catboost is removed because it has a `name` parameter, making it incompatible with hcrystalball
+                    if "catboost" in estimator_list:
+                        estimator_list.remove("catboost")
+                    if self._state.task in TS_FORECASTREGRESSION:
+                        try:
+                            import prophet
+
+                            estimator_list += ["prophet", "arima", "sarimax"]
+                        except ImportError:
+                            estimator_list += ["arima", "sarimax"]
+                elif "regression" != self._state.task:
+                    estimator_list += ["lrl1"]
+
+        if isinstance(starting_points, dict):
+            _estimators_from_starting_points = starting_points.keys()
+            if not any(i in estimator_list for i in _estimators_from_starting_points):
+                logger.warning(
```

Comment:
Please add a test to cover this case.

Reply:
@godcallray The code example posted in this issue: https://github.com/microsoft/FLAML/issues/795
could be used as a test for this case (just need to change the "time_budget" to a smaller number).

Reply:
I added a test accordingly.